test/conflictResolution.test.ts

```diff
@@ -181,11 +301,12 @@ replicationSuite('Conflict resolution', (appStateA: AppState, appStateB) => {
       .post('/api/v3/getUpdates')
```

Comment:
Add an assertion that the resolveUpdate timestamp does not contain a sub-version.